test/e2e/network/ingress.go

```diff
@@ -628,8 +628,10 @@ var _ = SIGDescribe("Loadbalancing: L7", func() {
 			}
 
 			ginkgo.By("Cleaning up cloud resources")
-			err := gceController.CleanupIngressController()
-			framework.ExpectNoError(err)
+			if gceController != nil {
+				err := gceController.CleanupIngressController()
+				framework.ExpectNoError(err)
+			}
```

Comment:
in the other `AfterEach`es in this file, they return after printing `"No ingress created, no cleanup necessary"`, so this one should just be fixed to do the same

Reply:
this test is skipped if is not running in gce or gke, is it possible that the gce.controller is nil if the Skip check succeeds?
```go
// Platform specific setup
ginkgo.BeforeEach(func() {
e2eskipper.SkipUnlessProviderIs("gce", "gke")
```
https://github.com/kubernetes/kubernetes/pull/92238/files#diff-ae2588ed31c4a644d59d239e3ed191dfR596-R598